cluster/charts/rook/values.yaml-e

```diff
@@ -0,0 +1,28 @@
+# Default values for rook-operator.
```

Comment:
I'm not following why this change is needed nor the weird extension to the file.

Reply:
i've seen this file get generated recently when i run `minikube.sh up` I believe, but i'm not sure what exactly is generating it.  @travisn probably accidentally committed it.  perhaps a .gitignore entry for it would be good.

Reply:
strange, not sure where this file came from. i'll remove it